### PR TITLE
Get away users widget

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/45323

## Description
Added a support of the GetAwayUsers script to run as a widget.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
Context output + WarRoom
warmoom
Empty
war_room_empty


Dashboard
Empty:
away_empty_dashboard
non empty
Screen Shot 2022-01-03 at 18 43 12

**Comment by tomneeman151293** [comment]
@evisochek Please approve this is the expected behaviour to solve your issue

File: Packs/ShiftManagement/ReleaseNotes/1_3_4.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+#### Dashboards
+##### Shift Management 
+- Added the **AwayUsers** widget to the dashboard.
+
+#### Scripts
+##### GetAwayUsers
+- Added support of using the script as a widget.
```
Comment (amshamah419) [inline Packs/ShiftManagement/ReleaseNotes/1_3_4.md:8]:
```suggestion
- Added support for using the script as a widget.
```

**Review by evisochek — APPROVED** [review]

**Comment by evisochek** [comment]
> @evisochek Please approve this is the expected behavior to solve your issue

From my side this behavior is approved and solves my issue

**Review by dantavori — APPROVED** [review]

**Pull request merged**